packages/fluentui/docs/src/examples/components/Form/Usage/FormExampleErrorAndSuccessful.shorthand.tsx

```diff
@@ -18,9 +17,9 @@ const fields = [
     key: 'last-name',
     control: {
       as: Input,
-      successIndicator: <PresenceAvailableIcon />,
+      showSuccessIndicator: true,
+      required: true,
```

Comment:
was it intentional to move it? then we do not show *

Reply:
Nops, by mistake. Fixed already